packages/main/src/plugin/index.ts

```diff
@@ -3164,19 +3164,22 @@ export class PluginSystem {
   }
 
   getLogHandler(channel: string, loggerId: string): LoggerWithEnd {
+    // Helper function to safely send to renderer, handling shutdown race condition
+    const safeSend = (messageType: string, data?: unknown): void => {
+      if (!this.isQuitting) {
+        try {
+          this.getWebContentsSender().send(channel, loggerId, messageType, data);
+        } catch (err) {
```

Comment:
I am afraid that this would silence other errors, it the check for isQuitting not enough that we have to add the try catch?